app/views/layouts/application.html.slim

```diff
@@ -39,12 +39,12 @@ html
         = render '/flash_notice_error'
         = yield
     javascript:
-      jQuery( () => {
+      jquery( function() {
```

Comment:
This goes in a browser - better not to use ES6 syntax because it's got better support.  Probably not a huge deal, but I saw it so I cleaned it up.